hardware/computemodule/schematics.md

```diff
@@ -1,14 +1,13 @@
 # Compute Module and related schematics
 
-Schematics for the Compute Module
+Schematics for the latest Compute Module 4 and the IO board can be found in the datasheets as follows:
```

Comment:
I would remove the word `latest` from here, since it implies that there was a previous CM4 before the latest one. (It's also not required).